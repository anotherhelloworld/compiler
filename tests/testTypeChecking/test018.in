var
    a: integer = 10;
begin
    a := 'a' xor 10;
end.
