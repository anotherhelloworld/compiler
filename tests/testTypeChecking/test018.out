Incompatible types, expected "integer" but "char" found in pos (4, 20)
